.github/workflows/move-issues-to-in-progress.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
             exit 1
           }
 
-          project_name="Issue backlog"
+          project_name="runtime-rs"
```

Comment:
Should we revert this back to `Issue backlog`?
/cc @jodh-intel @kata-containers/architecture-committee

Reply:
Good catch @liubin ! ;)